Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,76 @@ label monika_why_do_you_read:
+    m "In D&D, not all players are equal, since there necessarily always needs to be a 'Dungeon Master', basically someone who makes the setting of the story and controls the monsters and other obstacles."
+
+    m "Other Tabletop RPGs are a lot like theatre in the sense that everyone plays a character in the story, aside from the fact that the story itself is being improvised as it goes."
+    m "Everyone participating is taking turns, writing a story by reacting to the others' actions and acting on it."
+    m "I guess that makes it a good writing exercise..."
+    m "You don't need to think of the entire story by yourself, but at the same time, your character will end up in situations you probably never could anticipate..."
+    m "...So you have to improvise at all times by choosing what you think would be the best course of action in every situation, while still staying in-character."
+
+    m "Unlike most other forms of literature, it's not an author expressing their own thoughts and opinions through written text."
+    m "The best way I can explain it, is that tabletop roleplaying is kind of like the multiplayer experience of literature."
+    m "Just a little video game reference for you~"
+    m "The only problem with that though, is that once you reach the end of the story, if you even manage to get to a point in the story that you can call an ending..."
+    m "You'll often end up with a big, almost unreadable mess."
+    m "So it's harder to share this piece of literature with the world, which is one of my favorite parts about literature in general."
+    m "It kind of becomes a 'you need to have been there to get it' kind of story."
+    m "But the metafictional story of how a group of people came together to have a good time with a roleplaying game, may in fact inspire people around the world too."
+    m "It may even be even more of an epic than the story in novel form would make on its own, especially if you really get into it."
+    m "I will always prefer the beauty and wonder that real life can bring people when they do something fun together, after all."
+
+    if persistent._mas_pm_likes_board_games:
+        m "Maybe at some point we could make a little duo adventure or something...That would really make my day."
+        m "I'm happy just talking about it~"
+    else:
+        m "I know you don't play too many board games, so I don't expect you to join me in an adventure sometime, but that's okay!"
+        m "I'm happy just talking about it~"
+    if seen_event('monika_tabletop_rping'):
+        m 2eka "..."
+        m "You know..."
+        m 2rkd "I just realized, but it's a bit of a shame that I never thought of doing activities like that back in the club."
+        m 2ekb "We probably could've had a lot of fun together."
+        m 2eua "And who knows, maybe it would have made the club look more interesting to potential new members."
+        m 1ruc "Though, if I'm being honest... {w=0.8}{nw}"
```

Comment:
There's an extra space after the '...'. Also, I'm not certain about it but it's usualy {w=0.5} max for this kind of separations.